src/parameters/aoi.py

```diff
@@ -11,7 +11,7 @@ def aoi_parameter():
         parameterType="Required",
         direction="Input",
     )
-    param.filter.list = ["Point"]
+    param.filter.list = ["Point", "Polygon"]
```

Comment:
We used to not accept polygon for new tasking orders, is that still true? Maybe we make two separate aoi parameters in that case, one that contains polygons and can be used in the analytic ordering tool and then one with just points that's used in the tasking tool

Reply:
Resolving this based of conversation offline. We do now accept polygons in tasking so this should all be happy :)